src/runtime/database.h

```diff
@@ -159,7 +159,9 @@ struct Database {
   std::string get_hash(const std::string &file, long modified);
 
   // Outer is a set of filters to be AND'd together, inner is a set of queries to be OR'd together
```

Comment:
Probably should be updated to at least mention that outer/inner is referring to `core_filters`